lib/Service/AiIntegrations/AiIntegrationsService.php

```diff
@@ -115,4 +168,33 @@ public function isLlmAvailable(): bool {
 		}
 		return in_array(SummaryTaskType::class, $manager->getAvailableTaskTypes(), true);
 	}
+
+	private function isPersonalEmail(IMAPMessage $imapMessage): bool {
+		$commonPatterns = [
+			'noreply', 'no-reply', 'info', 'notification', 'donotreply', 'donot-reply',
+			'admin', 'support', 'contact', 'webmaster', 'helpdesk', 'sales', 'marketing',
```

Comment:
this might give false negatives. if you are contacted by support@domain.tld it might still be an email personalized to you and awaiting a response.

IMO when in doubt I'd rather show the suggested replies than hide them for legitimate messages.

Reply:
The I would suggest removing the first llm check. Cause it's usually a false negative

Reply:
Makes sense

Reply:
support@ and friends are still checked. If you receive an email from support asking if you have tried turning it off and on again the app should still suggest replies.

Let's drop the pattern matching

Reply:
Can we leave the basic ones?
`'noreply', 'no-reply', 'notification', 'donotreply', 'donot-reply','noreply-', 'do-not-reply', 'automated',
'donotreply-', 'noreply.', 'noreply_', 'do_not_reply', 'no_reply', 'no-reply', 
'automated-', 'do_not_reply', 'noreply+',`

Reply:
Sure